README.md

```diff
@@ -19,6 +19,13 @@ After downloading the zip from above:
 2. Follow the instructions in "install-mac.txt" or "install-win.txt", depending
    on what operating system you're using.
 
+Inter UI has also been packaged by many third parties:
```

Comment:
Name is "Inter" (sans "UI).
Can you change this line to:

"Font files are also available from the following third-party distributors:"